File: DSA/NamanDixit_500125539/Day3/Question2.java
import java.util.Scanner;
public class Question2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter length of array: ");
        int length = scanner.nextInt();
        int[] array = new int[length];
        System.out.println("Enter elements of array:");
        for (int i = 0; i < length; i++) {
            array[i] = scanner.nextInt();
        }
        System.out.println("Enter Target value: ");
        int target = scanner.nextInt();

        int count = 0;
        for (int i = 0; i < length; i++) {
            if (array[i] == target) {
                count++;
            }
        }
        System.out.println("Number of occurrences of " + target + ": " + count);
    }
    
}

File: DSA/MehakDilawar_590025015/Day_3/Question2.py
#Given an sorted integer array and a target value, count the number of occurrences of the target in the array and return the result.
#Input:[1,1,2,2,2,2] target:2. Output:4
def target_occurence(integer, target):
   count=0
   for i in range(len(integer)):
    if integer[i]==target:
        count+=1
   print(count)

integer=[]
n=int(input("Enter no. of elements: "))
for i in range(n):
    x=int(input("Enter element: "))
    integer.append(x)
    integer.sort()
target=int(input("Enter target value: "))
target_occurence(integer, target)

